Fix:

error: assignment of read-only member 'rapidjson::GenericStringRef::s'
error: assignment of read-only member 'rapidjson::GenericStringRef::length'

https://github.com/Tencent/rapidjson/issues/718
https://github.com/Tencent/rapidjson/commit/3b2441b87f99ab65f37b141a7b548ebadb607b96

https://github.com/BYVoid/OpenCC/issues/884
https://github.com/BYVoid/OpenCC/commit/e1f3e6b17ebd70c799926e675fe8d1ec72f6b36a
--- deps/rapidjson-1.1.0/rapidjson/document.h
+++ deps/rapidjson-1.1.0/rapidjson/document.h
@@ -316,8 +316,6 @@ struct GenericStringRef {
 
     GenericStringRef(const GenericStringRef& rhs) : s(rhs.s), length(rhs.length) {}
 
-    GenericStringRef& operator=(const GenericStringRef& rhs) { s = rhs.s; length = rhs.length; }
-
     //! implicit conversion to plain CharType pointer
     operator const Ch *() const { return s; }
 
